src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheck.java

```diff
@@ -60,6 +60,13 @@
  *   &lt;property name=&quot;format&quot; value=&quot;a href&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public void myTest() {
+ *     String test = "a href"; // violation
+ *
```

Comment:
Please add example that check is case-sensitive

Reply:
I am done.